Tests/ScipioKitTests/ProjectGeneratorTests.swift

```diff
@@ -8,26 +8,25 @@ private let fixturePath = URL(fileURLWithPath: #file)
     .appendingPathComponent("Resources")
     .appendingPathComponent("Fixtures")
 private let testPackagePath = fixturePath.appendingPathComponent("E2ETestPackage")
+private let clangPackagePath = fixturePath.appendingPathComponent("ClangPackage")
 
 final class ProjectGeneratorTests: XCTestCase {
-    private var projectGenerator: ProjectGenerator!
-    private var package: Package!
     private let fileSystem: some FileSystem = localFileSystem
 
-    override func setUpWithError() throws {
-        package = try Package(packageDirectory: testPackagePath)
-        projectGenerator = ProjectGenerator(package: package,
-                                            buildOptions: .init(buildConfiguration: .debug,
-                                                                isSimulatorSupported: true,
-                                                                isDebugSymbolsEmbedded: false,
-                                                                frameworkType: .static,
-                                                                sdks: [.iOS]),
-                                            fileSystem: localFileSystem)
-
-        try super.setUpWithError()
-    }
+    private func prepareGenerator(for packagePath: URL) throws -> (Package, ProjectGenerator) {
```

Comment:
I feel this function doing too much things, (not only initalize project generator but also loading package), how about receiving `Package` object, not package path?

Reply:
That makes sense.
But this is a test code. So I think it's better to avoid writing a same code twice.

Reply:
Fixed!